src/SDKs/RecoveryServices.Backup/Management.RecoveryServices.Backup/Microsoft.Azure.Management.RecoveryServices.Backup.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <PackageId>Microsoft.Azure.Management.RecoveryServices.Backup</PackageId>
     <Description>Provides developers with libraries for the updated recovery services backup feature under Azure Resource manager.</Description>
-    <VersionPrefix>1.3.0-preview</VersionPrefix>
+    <VersionPrefix>1.4.0-preview</VersionPrefix>
```

Comment:
@DheerendraRathor you have a breaking change, that requires major version change and not minor version.

Please also use latest resource manager nuget package for recording your tests that you have in this PR.
You are still using 1.1.0-preview Resource Manager and you need start using >=1.6.0